### DISCUSSION TITLE
Microphone mixer setting

### DISCUSSION THREAD

**Discussion by pjalocha** [mcguirepr89/BirdNET-Pi#359] [General]
How is the microphone mixer setting decided ?
Is it automatic or in the config file somewhere ?
I find I can change it manually and make the audio louder, but does it help anything ?
In general audio levels look fairly soft so to listen on the laptop I need to boost the volume.

**Comment by unosonic** [comment]
You can use `alsamixer` on the command line (per ssh or in web terminal), it should "remember" the changes you've made, at least it does on my Pi. If not, you can try the various possibilities described here: https://askubuntu.com/questions/50067/howto-save-alsamixer-settings

**Reply by ehpersonal38** [reply]
This is the recommended way, which is also described in the wiki: https://github.com/mcguirepr89/BirdNET-Pi/wiki/Adjusting-your-sound-card

I will add that if you pull up the live "Spectrogram" page at the same time you're fiddling with alsamixer, you can see in near-realtime the adjustments you make (wait about 10 seconds after each adjustment for the mic delay to catch up). This will let you play with the gain until you find a value that you're happy with.

**Comment by pjalocha** [comment]
Thank you, can you see other setting of the audio card like sampling rate and number of channels ?
For example I now see the spectrogram only goes to about 11kHz, does this mean the audio runs at 22kHz or is the spectrogram limit intentionally set so ?

When you change the mixer setting manually it is not remembered across reboots ?

is there any concrete aim when you change the gain, does it help at all when the gain is set higher, how can you judge if the system performance gets any better or worse ?

BTW: a background noise level plot would be useful to judge the microphone placement, for example when I try to avoid nearby road noise.
